app/transforms/utc.js

```diff
@@ -0,0 +1,12 @@
+import moment from 'moment';
+import DS from "ember-data";
+
+export default DS.Transform.extend({
+  serialize: function(value) {
+    return value ? value.toJSON() : null;
+  },
+
+  deserialize: function(value) {
+    return moment.utc(value);
```

Comment:
This request is based on the assumption that all `patient.birthDate` values are "incomplete", `YYYY-MM-DD`. Both my company's FHIR instances and `fhirtest.uhn.ca` do this but I have found no documentation to support the behavior. 

Your comment sparked a handful of questions in my head. At the very least `value` should be checked before any conversion. I'll get back to you.